### DISCUSSION TITLE
Required error on filled input

### DISCUSSION THREAD

**Discussion by tutods** [react-hook-form/react-hook-form#6187] [Q&A]
Hi Guys
I have an edit modal where I filled the inputs when open that modal.
But if I submit the form without touch all required fields the error message is show
<img width="509" alt="image" src="https://user-images.githubusercontent.com/18479474/128670302-18ed5697-68b8-4543-97ca-c5c737c23026.png">
**Note:** the message is in portuguese, but is the required error message.

Anyone already have this problem?

**Comment by jorisre** [comment]
Hi! Can you share a codesandbox with your issue, please?
